continuous-integration/linux/lint.sh

```diff
@@ -28,4 +28,17 @@ if [ $errorsFound -ne 0 ]; then
     echo "ERROR: $errorsFound formatting error(s) found! See the diffs for each file printed above."
     exit 1
 fi
+
+BUILD_DIR="$ROOT_DIR/build/ci/tidy"
+mkdir --parents "$BUILD_DIR" || exit $?
+cd "$BUILD_DIR" || exit $?
+cmake -GNinja \
+    -DUSE_EIGEN3=OFF \
+    -DUSE_OPENCV=OFF \
+    -DCMAKE_CXX_CLANG_TIDY="/usr/bin/clang-tidy-8" \
```

Comment:
I would bumpt to 9 and probably to 10 as well as it is going to be release before this PR is done.